src/pages/Settings/PaymentSettings.vue

```diff
@@ -0,0 +1,339 @@
+<template>
+	<www-page class="payments" :gray-background="true">
+		<the-my-kiva-secondary-menu slot="secondary" />
+		<div class="title-area">
+			<div class="row column">
+				<h1>
+					Payments Settings
+				</h1>
+			</div>
+		</div>
+		<div class="row">
+			<kv-settings-card class="columns small-12 large-8" title="Credit Card Settings">
```

Comment:
Do we also show the vaulted PayPal – or any other vaulted – payment methods? If so "Credit Card Settings" may not be accurate.

Reply:
We do not show any other payment methods. Paypal and GooglePay payments cannot be "saved" in the drop-in so they are never vaulted